packages/shared/src/stac/index.ts

```diff
@@ -1,5 +1,7 @@
-export const StacVersion = '1.0.0';
+export const StacVersion = '1.0.0-beta.2';
 export const StacLicense = 'CC-BY-4.0';
+export const StacBaseMapsExtension =
```

Comment:
Do we hard code `basemaps.linz.govt.nz` anywhere else? Ideally we should be injecting this later?

Could we make it a config variable inside the bundler?

Reply:
Do we know the host at run time for `cli `and `bathymetry` packaes and at deploy time for `landing`?

Reply:
I think, because this is a link to a LINZ specific schema, the well known location should always be used.

Reply:
> because this is a link to a LINZ specific schema

Ok good reasoning!